docs/subnets/network-runner.md

```diff
@@ -577,7 +581,7 @@ avalanche-network-runner control start \
 --endpoint="0.0.0.0:8080" \
 --avalanchego-path ${AVALANCHEGO_EXEC_PATH} \
 --plugin-dir ${AVALANCHEGO_PLUGIN_PATH} \
---custom-vms '{"blobvm":"/tmp/blobvm.genesis.json"}'
+--blockchain-specs '[{"vm_name":"blobvm","genesis":"/tmp/blobvm.genesis.json"}]' \
```

Comment:
probably you don't need the \ at the end

Reply:
fixed